src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs

```diff
@@ -208,6 +218,34 @@ protected ProductsPage SelectProducts(Product[] products)
                 });
         }
 
+        protected ThankYouPage PayWithPayexTrustly(Product[] products, Checkout.Option checkout = Checkout.Option.Anonymous)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6074055)